detectable/src/main/java/com/blackduck/integration/detectable/detectables/opam/parse/OpamFileParser.java

```diff
@@ -23,8 +23,10 @@ public class OpamFileParser {
     private static final String DEPENDS = "depends";
     private static final String NAME = "name";
     private boolean inVersionSection = false;
+    private boolean inDependsSection = false;
 
     public OpamFileParser() {
+        // We do not need any variables to be initialized so keeping it default
```

Comment:
if this is the case is there any value in explicitly declaring the default constructor? Maybe consider removing it entirely?

Reply:
Yeah will do